astron/darwin/start-ai-server.sh

```diff
@@ -7,8 +7,8 @@ export DYLD_FRAMEWORK_PATH="Frameworks"
 # Define some constants for our AI server:
 MAX_CHANNELS=999999
 STATESERVER=4002
-ASTRON_IP="127.0.0.1:7100"
-EVENTLOGGER_IP="127.0.0.1:7198"
```

Comment:
Why are you changing the ports? there's no good reason to change the ports to more obscure ports.

Reply:
Because astron cluster doesn’t use these ports . I changed the ports to ones windows used

Reply:
Otherwise the ai and uberdog can’t connect to astron

Reply:
ah, I see I thought you were changing the ports in another part my bad.